build-scripts/profile_tool.py

```diff
@@ -2,150 +2,246 @@
 
 from __future__ import print_function
 
-import json
 import argparse
-import jinja2
-import os
-import os.path
 
 try:
-    import ssg.build_profile
-    import ssg.constants
-    import ssg.environment
-    import ssg.xml
-    import ssg.build_yaml
-    from ssg.utils import mkdir_p
+    from utils.profile_tool import command_stats, command_sub
 except ImportError:
     print("The ssg module could not be found.")
-    print("Run .pyenv.sh available in the project root diretory,"
-          " or add it to PYTHONPATH manually.")
+    print(
+        "Run .pyenv.sh available in the project root diretory,"
+        " or add it to PYTHONPATH manually."
+    )
     print("$ source .pyenv.sh")
     exit(1)
 
 
-def parse_args():
-    script_desc = \
-        "Obtains and displays XCCDF profile statistics. Namely number " + \
-        "of rules in the profile, how many of these rules have their OVAL " + \
-        "check implemented, how many have a remediation available, ..."
-
-    parser = argparse.ArgumentParser(description="Profile statistics and utilities tool")
-    subparsers = parser.add_subparsers(title='subcommands', dest="subcommand")
-    parser_stats = subparsers.add_parser("stats", description=script_desc,
-                                           help=("Show profile statistics"))
-    parser_stats.add_argument("--profile", "-p",
-                        action="store",
-                        help="Show statistics for this XCCDF Profile only. If "
-                        "not provided the script will show stats for all "
-                        "available profiles.")
+def parse_stats(subparsers):
+    parser_stats = subparsers.add_parser(
+        "stats",
+        description=(
+            "Obtains and displays XCCDF profile statistics. Namely number "
```

Comment:
According or our Style Guides, the limit is 99 instead of 80. We can save some lines with these descriptions.
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#all-files